packages/base/src/3dview/mainview.tsx

```diff
@@ -227,7 +227,8 @@ export class MainView extends React.Component<IProps, IStates> {
 
       this._scene.add(new THREE.AmbientLight(0xffffff, 0.5)); // soft white light
 
-      this._cameraLight = new THREE.PointLight(0xffffff, 1);
+      this._cameraLight = new THREE.PointLight(0xffffff, 100);
+      this._cameraLight.decay = 5;
```

Comment:
I thought we would be using decay `0` with an intensity of `1`?

Reply:
Oh Shoot 😬 , this is my bad I think i pushed the wrong changes as I was changing them rapidly. So Sorry

Reply:
No worries!